libqtile/backend/base.py

```diff
@@ -158,6 +158,13 @@ def hide(self) -> None:
     def unhide(self) -> None:
         """Unhide the window"""
 
+    def is_visible(self) -> bool:
+        """Is this window visible (i.e. not hidden)?"""
+        return False
+
+    def cmd_is_visible(self) -> bool:
+        return self.is_visible()
```

Comment:
These will need to be combined when commands get decorated, so maybe we should just combine them here?

Reply:
Yep, done, thanks.